scripts/build-engine/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cocos/build-engine",
-  "version": "4.4.1-alpha.10",
+  "version": "4.4.1-oh.0",
```

Comment:
@PPpro could you please take a look this modification? Thanks.

Reply:
sure, it's OK here

Reply:
ok